src/skai/model/README.md

```diff
@@ -61,3 +61,19 @@ gs://path/to/dataset
 ```
 
  b. `--project_path=~/path/to/skai` - Here, provide the entrypoint to the cloned skai project. for instance `/home/user/my_projects/skai`.
+
+ c. The `--accelerator` and `--accelerator_count` flags provides the options to choose an accelerator type and the number of this accelerator to run experiments on. 
+
+ ```
+ # Example
+    # Run experiment on 8 number of V100 GPU machines
```

Comment:
This means each trial will use 8 V100 GPUs. Use 1 for this example instead or remove it so that it uses the default value which is 1.

Add another example for TPUs and specify that they user must specify accelerator_count=8